docs: add BitTasker Wallet SDK documentation and update dev-tools index (#514)
rsksmart/devportal: _rsk/rbtc/gas.md

The `minimumGasPrice` is written in the block header by miners and establishes the minimum gas price a transaction should have in order to be included in that block.
Negotiated minimum gas price is described in [RSKIP09](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP09.md).

> Minimum gas price is written in the block header by miners. It describes the minimum gas price a transaction should have to be included in that block. It can change with time, by up 1% of the `minimumGasPrice` of the previous block.

Reviewer: ```suggestion
Minimum gas price is written in the block header by miners. It describes the minimum gas price a transaction should have to be included in that block. It can change with time, by up to 1% of the `minimumGasPrice` of the previous block.
```

The latest block's minimum gas price can be obtained using this Web3 method:

```javascript
web3.eth.getBlock('latest').minimumGasPrice
```

> There are some practical options about this subject:

Author: ```suggestion
Here are some practical approaches to this subject:
```
